extensions/localserver-cli-command/src/module-loader/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "@oracle/module-loader",
+    "version": "1.0.0",
+    "description": "NS module-loader",
+    "scripts": {
+        "test": "jest --config module-loader.jest.config.js module-loaderTest.js",
+        "test:debug": "node --inspect node_modules/jest/bin/jest --config module-loader.jest.config.js module-loaderTest.js --runInBand"
+    },
+    "main": "module-loader.js",
+    "author": "",
+    "dependencies": {},
+    "devDependencies": {
+        "jest": "25.5.1"
```

Comment:
I have filled and OSSR for  jest version 26.6.3, and it was approved. You could use it if you want.